package.json

```diff
@@ -125,7 +124,7 @@
     "start": "export NODE_ENV=development; check-engines && check-dependencies && webpack-dashboard -p 3736 -- webpack-dev-server --config ./webpack.dev.js",
     "_build": "export HEAD_COMMIT=$(git rev-parse --short HEAD); check-engines && check-dependencies && rimraf dist; webpack --config ./webpack.build.js --progress --color",
     "build-production": "export NODE_ENV=production; npm run _build",
-    "serve-static": "export NODE_ENV=staging; check-engines && check-dependencies && npm run _build && node ./static-server.js",
+    "serve-static": "npm ci; export NODE_ENV=production; check-engines && check-dependencies  && npm run _build && node ./static-server.js",
```

Comment:
Noting here that `staging` is an invalid value for `NODE_ENV` in modern Webpack builds. See https://webpack.js.org/configuration/mode/.